src/nxdoc/web-ui/web-ui-tutorials/web-ui-search.md

```diff
@@ -1,40 +1,44 @@
 ---
 title: 'HOWTO: Configure Searches'
+description: Learn how to create a new search screen with an icon in the left menu. The search will be compose of a full text search, a search by date range, a search by owners and finally by tags.
 review:
-    comment: ''
-    date: '2018-02-21'
-    status: ok
+  comment: ''
+  date: '2019-02-18'
+  status: ok
 toc: true
 details:
-    howto:
-        excerpt: Learn how to create a new search screen with Studio Designer.
-        level: Intermediate
-        tool: 'Studio Designer, code'
-        topics: 'Web UI, Studio Designer'
+  howto:
+    excerpt: Learn how to create a new search screen with Studio Designer.
+    level: Intermediate
+    tool: 'Studio Designer, code'
+    topics: 'Web UI, Studio Designer'
 labels:
-    - lts2016-ok
-    - nuxeo-web-ui
-    - tcardoso
-    - extension
-    - university
-    - search-tab-component
-    - lts2017-ok
+  - lts2016-ok
+  - nuxeo-web-ui
+  - tcardoso
+  - extension
+  - university
+  - search-tab-component
+  - lts2017-ok
 tree_item_index: 400
-
 ---
+
 After having created your own [document type]({{page page='web-ui-document-layouts'}}#how-to-define-a-new-document-type-layout-with-the-view-designer) you will probably want to create your own search.
 
-We will learn how to create a new search screen with an icon in the left menu. The search will be compose of a full text search, a search by date range, a search by owners and finally by tags.
+{{! excerpt}}
+Learn how to create a new search screen with an icon in the left menu. The search will be compose of a full text search, a search by date range, a search by owners and finally by tags.
+{{! /excerpt}}
 
 {{#> callout type='info'}}
```

Comment:
This callout should be updated so that the format is not broken the day we add screenshots from the intranet.